File: SDKs/Aspose.Pdf-Cloud-SDK-for-.NET/src/Com/Aspose/PDF/Model/TextItem.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class TextItem {
    public string Text { get; set; }

    public TextFormat Format { get; set; }

    public List<Link> Links { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class TextItem {\n");
      sb.Append("  Text: ").Append(Text).Append("\n");
      sb.Append("  Format: ").Append(Format).Append("\n");
      sb.Append("  Links: ").Append(Links).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Pdf-Cloud-SDK-for-Java/src/main/java/com/aspose/pdf/model/TextItem.java
package com.aspose.pdf.model;

import java.util.*;
import com.aspose.pdf.model.TextFormat;
public class TextItem {
  private String Text = null;
  private TextFormat Format = null;
  private List<Link> Links = new ArrayList<Link>();
  /**
	 * getText
	 * Gets String
	 * @return Text
	 */
  public String getText() {
    return Text;
  }

	/**
	 * setText
	 * Sets String
	 * @param Text String
	 */
  public void setText(String Text) {
    this.Text = Text;
  }

  /**
	 * getFormat
	 * Gets TextFormat
	 * @return Format
	 */
  public TextFormat getFormat() {
    return Format;
  }

	/**
	 * setFormat
	 * Sets TextFormat
	 * @param Format TextFormat
	 */
  public void setFormat(TextFormat Format) {
    this.Format = Format;
  }

  /**
	 * getLinks
	 * Gets List<Link>
	 * @return Links
	 */
  public List<Link> getLinks() {
    return Links;
  }

	/**
	 * setLinks
	 * Sets List<Link>
	 * @param Links List<Link>
	 */
  public void setLinks(List<Link> Links) {
    this.Links = Links;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class TextItem {\n");
    sb.append("  Text: ").append(Text).append("\n");
    sb.append("  Format: ").append(Format).append("\n");
    sb.append("  Links: ").append(Links).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

